src/test/java/com/redhat/engineering/srcclr/SourceClearTest.java

```diff
@@ -47,7 +48,7 @@ public void runSourceClear() throws Exception
         catch( Exception e )
         {
             logger.debug( "Caught exception executing the wrapper", e );
-            throw e;
+            Assert.fail();
```

Comment:
I  think perhaps we should use `Assert.fail(  "<message>" ) ` in order to propagate out the information. And maybe change that debug line to error?